lexicons/com/atproto/server/getServiceAuth.json

```diff
@@ -0,0 +1,33 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.server.getServiceAuth",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get a signed token on behalf of the requesting DID for the requested service.",
+      "parameters": {
+        "type": "params",
+        "required": ["aud"],
+        "properties": {
```

Comment:
what about expiration (`exp`)? should client be allowed to request longer/shorter tokens within some range?

Reply:
Makes me a bit uneasy honestly - I don't think we want to offer long versions of these tokens. If you request one, you should really only be using it once

Reply:
I definitely think the returned token should have some `exp`; I guess we can just set it to 5-10min in the future or something like that?

Reply:
Oh to clarify - the returned token does have an `exp` and it's set to 60s 👍 

We just don't provide optionality around requesting a longer one